pareto/strategic_water_management/strategic_produced_water_optimization.py

```diff
@@ -3694,318 +2819,44 @@ def TotalReuseVolumeRule(model):
     )
 
     def PipingCostRule(model, l, l_tilde, t):
-        if l in model.s_PP and l_tilde in model.s_CP:
-            if model.p_PCA[l, l_tilde]:
-                constraint = (
-                    model.v_C_Piped[l, l_tilde, t]
-                    == model.v_F_Piped[l, l_tilde, t] * model.p_pi_Pipeline[l, l_tilde]
-                )
-                return process_constraint(constraint)
-            else:
-                return Constraint.Skip
-        elif l in model.s_PP and l_tilde in model.s_N:
-            if model.p_PNA[l, l_tilde]:
+        if (l, l_tilde) in model.s_LLA:
+            if l in model.s_F:
                 constraint = (
                     model.v_C_Piped[l, l_tilde, t]
-                    == model.v_F_Piped[l, l_tilde, t] * model.p_pi_Pipeline[l, l_tilde]
+                    == model.v_F_Sourced[l, l_tilde, t]
```

Comment:
The documentation shows FPiped and FSourced added together, which does not seem to match this implementation:

![image](https://user-images.githubusercontent.com/26798258/228701453-4380d60a-beb0-4b0d-acf4-dd9c90371e54.png)

Are the implementation and documentation actually equivalent, even though they appear at first glance not to be?

Reply:
yes, the documentation and implementation are actually same, this is the closest way i could represent the implemented constraint.